toucan_data_sdk/utils/generic/date_requester.py

```diff
@@ -38,14 +41,14 @@ def date_requester_generator(
         - key (*str*): name of the granularity
         - value (*str*): Format of the granularity e.g. '%d/%m/%Y' (see [pandas doc](
             https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior))
-    - `others_format` (*dict*) : Add new columns for each key
-        - key (*str*) : name of the column
+    - `others_format` (*dict*): Add new columns for each key
+        - key (*str*): name of the column
         - value (*str*): format of the granularity e.g. '%d/%m/%Y' (see [pandas doc](
             https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior))
-    - `times_delta` (*dict*) : Add new columns for each key
-        - key (*str*) : name of the column
+    - `times_delta` (*dict*): Add new columns for each key
+        - key (*str*): name of the column
         - value (*str*): time delta (e.g. '+1 day', '+3 day', '-4 month')
-
+    - `locale` (*str*): locale used to display the dates (by default, the client one is taken)
```

Comment:
"client one" doesn't seem meaningful in this context, does it? May the "process" one?

Reply:
👍